api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -49,13 +54,23 @@ public void willStart() {
      */
     public void started() {
 		LocationUtils.createPersonAttributeTypeForLocation();
+		generateOpenMRSPrivilegeForLocations();
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME,
 			    LocationBasedAccessConstants.PERSONATTRIBUTETYPE_UUID);
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
 			    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
 	    log.info("Location Based Access Control Module started");
     }
 
+	private void generateOpenMRSPrivilegeForLocations() {
```

Comment:
why we need OpenMRS in this method name? Let's go for a generic name

Reply:
I think `generatePrivilegesForLocations` would be good.

Reply:
Have you updated this PR with this name?

Reply:
Yes Updated.